Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Button.cs

```diff
@@ -179,6 +187,19 @@ protected override void HandleUpdate()
             }
         }
 
+        /*
```

Comment:
cool, at least it's a starting point if someone else gets round to looking at it first